lib/config.go

```diff
@@ -155,18 +179,14 @@ func parseKeys(cfg interface{}, prefix, parentKey string) ([]byte, error) {
 	return nil, fmt.Errorf("error getting %s from config", prefix)
 }
 
-// SetConfig validates, updates and saves the config
-func (m *ConfigMethods) SetConfig(ctx context.Context, update *config.Config) (*bool, error) {
+// SetConfig validates, updates, and saves the config
+func (configImpl) SetConfig(scope scope, update *config.Config) (*bool, error) {
 	res := false
-	if m.inst.http != nil {
-		return &res, ErrUnsupportedRPC
-	}
-
 	if err := update.Validate(); err != nil {
 		return &res, fmt.Errorf("validating config: %w", err)
 	}
 
-	if err := m.inst.ChangeConfig(update); err != nil {
+	if err := scope.Loader().ChangeConfig(update); err != nil {
```

Comment:
This is okay, just wanted to mention that this will be changing soon as per this TODO: https://github.com/qri-io/qri/issues/1661. Writing any of our data structures should be pulled up into `dispatch` such that it can be safely done in a multi-process environment.